### ISSUE TITLE
2019.46.4.0 - Freezing

### ISSUE THREAD

**Issue by natrlhy** [NateMccomb/TeslaCamViewerII#7]
![Annotation 2019-11-14 201623](https://user-images.githubusercontent.com/19696424/68916642-b6c38600-071b-11ea-8df2-d0889a3ddda8.png)
Manually updated to 2019.46.4.0. UI hangs badly when playing in faster speeds causing me to forcefully close the UI.

**Comment by NateMccomb** [comment]
I think it's something in the VLC ActiveX plugin but I haven't found what exactly is causing this issue yet. Maybe check windows event viewer around that time and see if spot anything that might be helpful.

**Comment by Kumasasa** [comment]
UI froze here without changing the playback speed, but I was clicking around in the playback window
 Got this:

```
- <Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
- <System>
  <Provider Name="Windows Error Reporting" /> 
  <EventID Qualifiers="0">1001</EventID> 
  <Level>4</Level> 
  <Task>0</Task> 
  <Keywords>0x80000000000000</Keywords> 
  <TimeCreated SystemTime="2019-11-15T07:35:03.356763100Z" /> 
  <EventRecordID>6480</EventRecordID> 
  <Channel>Application</Channel> 
  <Computer>Kalahari.kumasasa.local</Computer> 
  <Security /> 
  </System>
- <EventData>
  <Data>1546819983561925837</Data> 
  <Data>5</Data> 
  <Data>AppHangB1</Data> 
  <Data>Nicht verfügbar</Data> 
  <Data>0</Data> 
  <Data>TeslaCam Viewer II.exe</Data> 
  <Data>2019.46.4.0</Data> 
  <Data>c857f1ed</Data> 
  <Data>f0e6</Data> 
  <Data>134234628</Data> 
  <Data /> 
  <Data /> 
  <Data /> 
  <Data /> 
  <Data /> 
  <Data>\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER12A.tmp.WERInternalMetadata.xml \\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER14A.tmp.xml \\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER159.tmp.csv \\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER169.tmp.txt \\?\C:\Users\Ivo\AppData\Local\Temp\WER130E.tmp.appcompat.txt</Data> 
  <Data>\\?\C:\ProgramData\Microsoft\Windows\WER\ReportArchive\AppHang_TeslaCam Viewer _a68b47f7508c41a5a1a5937914bbab10c64951_272365b0_9d5be834-5e13-47ed-b1dc-5f2f38f7f2fe</Data> 
  <Data /> 
  <Data>0</Data> 
  <Data>1105e323-5a79-438d-84e1-cb884e0176b6</Data> 
  <Data>268435456</Data> 
  <Data>df0a4f8fde9b6516b577689463af24cd</Data> 
  <Data>0</Data> 
  </EventData>
  </Event>
```
and this

```
- <Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
- <System>
  <Provider Name="Application Hang" /> 
  <EventID Qualifiers="0">1002</EventID> 
  <Level>2</Level> 
  <Task>101</Task> 
  <Keywords>0x80000000000000</Keywords> 
  <TimeCreated SystemTime="2019-11-15T07:35:03.488685800Z" /> 
  <EventRecordID>6481</EventRecordID> 
  <Channel>Application</Channel> 
  <Computer>Kalahari.kumasasa.local</Computer> 
  <Security /> 
  </System>
- <EventData>
  <Data>TeslaCam Viewer II.exe</Data> 
  <Data>2019.46.4.0</Data> 
  <Data>1c74</Data> 
  <Data>01d59b866de780f3</Data> 
  <Data>11</Data> 
  <Data>G:\TeslaCam Viewer II.exe</Data> 
  <Data>1105e323-5a79-438d-84e1-cb884e0176b6</Data> 
  <Data /> 
  <Data /> 
  <Data>Cross-thread</Data> 
  <Binary>430072006F00730073002D00740068007200650061006400000044006500610064006C006F0063006B0000000000</Binary> 
  </EventData>
  </Event>
```

**Comment by NateMccomb** [comment]
Looks like I found someone with a similar issue due to commanding an already stopped player to "Stop" that can cause the VLC ActiveX to crash: https://github.com/ZeBobo5/Vlc.DotNet/issues/155

Checking if a VLC player is "Playing" or "Paused" before issuing a "Stop" command seems to have fixed this crashing issue. I have updated and fixed the code for this in 2019.46.5.1. Let me know if this fixed the problems you were seeing.
